dtsync/publish.go

```diff
@@ -60,6 +70,25 @@ func NewPublisher(host host.Host, ds datastore.Batching, lsys ipld.LinkSystem, t
 	}, nil
 }
 
+func NewPublisherWithTopic(host host.Host, ds datastore.Batching, lsys ipld.LinkSystem, t *pubsub.Topic) (*publisher, error) {
```

Comment:
Why is this needed if `NewPublisher` takes a `pubsub.Topic` as an option?

Reply:
my hack, I forgot to remove